bot.py

```diff
@@ -34,6 +34,9 @@ def substitute(update: Update, context: CallbackContext):
             useflags = re.IGNORECASE
         if 'm' in flags.lower():
             useflags |= re.MULTILINE
+        
```

Comment:
This is a different change, it hardcodes a check for s/.*/thing/ because that had weird behaviour